ui/src/notification/NotificationForm.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-container>
+    <v-row
+      cols="30"
+      sm="6"
+      md="4"
+    >
+      <v-select
+        v-model="notification.type"
+        :items="notificationTypes"
+        label="Type"
+        @change="resetParameters()"
+      />
+    </v-row>
+    <v-row
+      cols="20"
+      sm="6"
+      md="4"
+    >
+      <v-text-field
+        v-model="notification.condition"
+        label="Condition"
+      />
+    </v-row>
+    <v-row
+      cols="10"
+      sm="6"
+      md="4"
+    >
+      <webhook-notification-form
+        v-if="notification.type === 'webhook'"
+        v-model="notification.parameters"
+        style="flex: 1 1 auto"
+        @validityChanged="isValid = $event"
```

Comment:
not sure if that really works. Better separate and join form validation of this component with parameters via &&